x-pack/legacy/plugins/monitoring/server/lib/metrics/__test__/__snapshots__/metrics.test.js.snap

```diff
@@ -871,11 +1363,11 @@ Object {
     "label": "Too large",
     "metricAgg": "max",
     "timestampField": "beats_stats.timestamp",
-    "title": "Response Errors",
+    "title": "Response Errors Intake API",
```

Comment:
Should this be `Too Large` (comparing with the other titles).

Reply:
Thanks! Fixed.